src/BSON/Binary.c

```diff
@@ -130,9 +134,15 @@ static zend_function_entry php_phongo_binary_me[] = {
 
 
 /* {{{ php_phongo_binary_t object handlers */
-static void php_phongo_binary_free_object(void *object TSRMLS_DC) /* {{{ */
+#if PHP_VERSION_ID >= 70000
+static inline php_phongo_binary_t * php_phongo_binary_t_fetch_object(zend_object *obj)
+{
+      return (php_phongo_binary_t *)((char *)obj - XtOffsetOf(php_phongo_binary_t, std));
+}
+
```

Comment:
I did this already, see ./php_phongo_structs-7.h

You can now fetch the object using Z_BINARY_OBJ_P() (both in PHP5 and PHP7)